### PR TITLE
[Progress bar] indicating background color is incorrect when data-percent have decimal number

### PR DESCRIPTION
**Pull request by dutrieux**
## Description
[Progress bar] indicating `background-color` is incorrect when `data-percent` have decimal number.
This problem only occur when the data-number have decimal and under 10.

## Testcase
https://jsfiddle.net/dutrieux/xoz2L4sw/

## Screenshot
bug

### CODE REVIEW

File: src/definitions/modules/progress.less [hunk 1]
```diff
@@ -185,40 +185,40 @@
...
-  .ui.indicating.progress[data-percent="1"] .label,
-  .ui.indicating.progress[data-percent="2"] .label,
-  .ui.indicating.progress[data-percent="3"] .label,
-  .ui.indicating.progress[data-percent="4"] .label,
-  .ui.indicating.progress[data-percent="5"] .label,
-  .ui.indicating.progress[data-percent="6"] .label,
-  .ui.indicating.progress[data-percent="7"] .label,
-  .ui.indicating.progress[data-percent="8"] .label,
-  .ui.indicating.progress[data-percent="9"] .label {
+  .ui.indicating.progress[data-percent="0"] .label  .ui.indicating.progress[data-percent^="0."] .label,
```
Comment (lubber-de) [inline src/definitions/modules/progress.less:199]: A comma is missing here

Comment (dutrieux) [reply src/definitions/modules/progress.less:199]: The good method to solve the comma problem to Github is to commit on this commit or rebase ...

Comment (lubber-de) [reply src/definitions/modules/progress.less:199]: If we have the coice, then rebase 😄 .

Comment (dutrieux) [reply src/definitions/modules/progress.less:199]: I'm confuse I don't know what to do :
image

Sorry I'am very null with git :s

Comment (lubber-de) [reply src/definitions/modules/progress.less:199]: Some PRs have been merged into develop since you created your branch.
Just merge, it should be fine

Comment (dutrieux) [reply src/definitions/modules/progress.less:199]: I rebase locally the two commit (first + comma) with comma commit with squash mode

image

but when I push on github I get this previous message

Sorry I'am very null with git :s

Comment (lubber-de) [reply src/definitions/modules/progress.less:199]: Click on the "Merge" button ;)

Comment (dutrieux) [reply src/definitions/modules/progress.less:199]: I do it, thx @lubber-de

**Pull request merged**